apollo-backend/src/main/java/io/logz/apollo/services/BlockerService.java

```diff
@@ -188,4 +205,18 @@ private boolean isBlockerInScope(Blocker blocker, Deployment deployment) {
 
         return false;
     }
+
+    public synchronized BlockerFunctionResponse checkDeploymentShouldBeBlockedByServiceByRegionBlocker(List<Integer> serviceIds, List<Integer> environmentIds) {
+        for (Blocker blocker : getSingleRegionBlockers()) {
+            BlockerFunctionResponse singleRegionBlockerResponse = blocker.getBlockerFunction().shouldBlock(
+                    new RequestBlockerArguments(blockerInjectableCommons, serviceIds, environmentIds, blocker.getAvailability()));
+
+            if (((SingleRegionBlockerFunctionResponse)singleRegionBlockerResponse).isShouldBlock()){
```

Comment:
missing space in `(SingleRegionBlockerFunctionResponse) singleRegionBlockerResponse`